packages/concerto-cli/index.js

```diff
@@ -20,13 +20,18 @@ const Commands = require('./lib/commands');
 
 require('yargs')
     .scriptName('concerto')
-    .usage('$0 <cmd> [args]')
+    .usage('$0 <cmd> [manager] [args]')
```

Comment:
This change to `usage` is inconsistent with the changes below which use a `--ctoSystem` option as part of the arguments.